src/HasLabelCollection.php

```
@@ -0,0 +1,229 @@
+     * ```php
+     * $component->getLabelAttribute('id', 'default-id');
+     * ```
+     *
+     * @param string|UnitEnum $key Attribute name.
+     * @param mixed $default Default value when the attribute is missing.
+     *
+     * @return mixed Attribute value, or `$default` when missing.
+     */
+    public function getLabelAttribute(string|UnitEnum $key, mixed $default = null): mixed
+    {
+        $normalizedKey = Enum::normalizeValue($key);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Enum' in method 'getLabelAttribute'. (undefined)

(StaticAccess)